integration_tests/test_data.py

```diff
@@ -85,6 +89,13 @@ def render_json_template(env, template_file, **kwargs):
 TILESPECS_LC_JSON = render_json_template(example_env, 'test_LC.json',
                                          test_data_root=TEST_DATA_ROOT)
 
+ROUGH_MONTAGE_TILESPECS_JSON = render_json_template(example_env, 'rough_montage_tilespecs.json',
```

Comment:
this file is missing

Reply:
removed this line as is irrelevant to em montage qc.